rocAL/include/loaders/audio/node_audio_loader.h

```diff
@@ -44,11 +44,16 @@ class AudioLoaderNode : public Node {
     /// \param load_batch_count Defines the quantum count of the Audios to be loaded. It's usually equal to the user's batch size.
     /// \param mem_type Memory type, host or device
     /// \param meta_data_reader Determines the meta-data information
+    /// \param last_batch_policy Determines the handling of the last batch when the shard size is not divisible by the batch size. Check RocalLastBatchPolicy() enum for possible values.
+    /// \param pad_last_batch_repeated If set to True, pads the shards last batch by repeating the last sample's data (dummy data).
+    /// \param stick_to_shard Determines whether reader should stick to a single shards dataset or it to be used in a round robin fashion.
+    /// \param shard_size Provides the size of the shard for an epoch.
     /// The loader will repeat Audios if necessary to be able to have Audios in multiples of the load_batch_count,
     /// for example if there are 10 Audios in the dataset and load_batch_count is 3, the loader repeats 2 Audios as if there are 12 Audios available.
     void Init(unsigned internal_shard_count, unsigned cpu_num_threads, const std::string &source_path,
               const std::string &file_list_path, StorageType storage_type, DecoderType decoder_type, bool shuffle, bool loop,
-              size_t load_batch_count, RocalMemType mem_type, std::shared_ptr<MetaDataReader> meta_data_reader);
+              size_t load_batch_count, RocalMemType mem_type, std::shared_ptr<MetaDataReader> meta_data_reader,
+              RocalBatchPolicy last_batch_policy = RocalBatchPolicy::FILL, bool last_batch_padded = false, bool stick_to_shard = false, signed shard_size = -1);
```

Comment:
Rename the last_batch_padded here to pad_last_batch_repeated. Comments are updated with correct name. Applies to all loader files